content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -65,12 +65,11 @@ To configure IPv4/IPv6 dual-stack, set dual-stack cluster network assignments:
 * kube-proxy:
   * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>`
 * kubelet:
-  * when there is no `--cloud-provider` the administrator can pass a comma-separated pair of IP
-    addresses via `--node-ip` to manually configure dual-stack `.status.addresses` for that Node.
-    If a Pod runs on that node in HostNetwork mode, the Pod reports these IP addresses in its
-    `.status.podIPs` field.
-    All `podIPs` in a node match the IP family preference defined by the `.status.addresses`
-    field for that Node.
```

Comment:
I ended up dropping this part because it's not really about `kubelet --node-ip` (it's true whether the IPs were assigned by `--node-ip` or by default) and because it's kind of obvious anyway... It was interesting to note these things at one point in the past when we had just _fixed_ that behavior, but at this point, this is just how anyone would _expect_ it to work.